packages/@headlessui-react/src/components/transition/transition.test.tsx

```diff
@@ -343,6 +343,7 @@ describe('Setup API', () => {
           <div
             class="foo1
         foo2 foo1 foo2 leave"
+            style=""
```

Comment:
Is this required? 🤔

Reply:
Yes, it's a side effect of rendering and our changing the style prop.

Reply:
Correction: it's not a side effect of JSDOM, but it happens in the browser as well. We can delete the `style` attribute but then we have to verified that it is not empty.

Reply:
Gotcha! 👍